neuromation/cli/job.py

```diff
@@ -376,6 +376,11 @@ def job() -> None:
             )
 
         click.echo("Press ^C to stop forwarding")
+        try:
+            await asyncio.sleep(float('inf'))
```

Comment:
@asvetlov please correct me to use a better infinity-waiting code

Reply:
Strictly speaking, it is not easy with current asyncio.
More or less robust way in the explicit SIGINT/SIGTERM handling.

Anyway, if you want to support Windows for Python 3.6 and 3.7 please replace infinite sleep with infinite loop of one-second periods of sleeping.